lib/ttnt/tasks.rb

```diff
@@ -0,0 +1,54 @@
+require 'rake/testtask'
+require 'rake/file_list'
+require 'ttnt/testtask'
+require 'ttnt/test_selector'
+require 'rugged'
+
+# Reference: https://github.com/bundler/bundler/blob/master/lib/bundler/gem_helper.rb
+module TTNT
+  class TaskHelper
+    include Rake::DSL if defined? Rake::DSL
+
+    def self.install_tasks
+      new.install
+    end
+
+    def install
+      namespace :ttnt do
+        desc 'Generate test-to-code mapping for current commit object'
+        task 'anchor' do
+          # TODO: what if multiple test tasks are defined?
+          test_task = TTNT::TestTask.instances.first
+          test_files = []
+          test_files += test_task.test_files.to_a if test_task.test_files
+          test_files += Rake::FileList[test_task.pattern] if test_task.pattern
+
+          # TODO: properly regard run options defined for Rake::TestTask
+          gem_root = File.expand_path('..', File.dirname(File.expand_path(__FILE__)))
+          args = "-I#{gem_root} -r ttnt/anchor"
+          args += " -I#{test_task.libs.join(':')}" unless test_task.libs.empty?
```

Comment:
Thank you! I fixed it here: https://github.com/Genki-S/ttnt/commit/4ce34aeb8267dc13dbbc625fd9315fbc06e00040

I thought `$LOAD_PATH` might be reset in subprocess, so I used `"-I#{gem_root}"` to access TTNT from within child ruby process. But I confirmed in my environment it worked without adding this gem's root to `$LOAD_PATH` like I did. However, I still don't know exactly how `$LOAD_PATH` in subprocesses behaves... Could you let me know some learning materials if you know any? (Google search and experimentation with pry did not get me to the result...)